Makefile

```diff
@@ -10,6 +10,9 @@ build:
 test:
 	go test ./...
 
+test-race:
+	go test ./... -race
```

Comment:
I personally like to have verbose logs when running tests using the `-v` flag.
Not mandatory, but it helps sometime.